### PR TITLE
presentation-api idlharness update

### PR DESCRIPTION
**Pull request by louaybassbouss**
update idlharness.html with latest Presentation API interfaces from the ED.

### CODE REVIEW

**Comment by louaybassbouss** [comment]
@zqzhang can you review and merge. I created a snapshot from the ED but I think there are minor changes comparing to the current idlharness.

**Comment by Honry** [comment]
@louaybassbouss , please fix the lint error. 
`INDENT TABS:presentation-api/receiving-ua/idlharness.html`

File: presentation-api/controlling-ua/idlharness.html [hunk 1]
```diff
@@ -36,15 +36,16 @@
 
 [Constructor(DOMString url)]
```
Comment (Honry) [inline presentation-api/controlling-ua/idlharness.html]: I just check the latest spec at [Interface PresentationRequest](https://w3c.github.io/presentation-api/#idl-def-presentationrequest), the constructor has been changed to `[Constructor(DOMString url),
 Constructor(DOMString[] urls)]`

**Comment by zqzhang** [comment]
Thanks @Honry for the review. @louaybassbouss please drop me a note when you address the comments above.

**Comment by louaybassbouss** [comment]
Done. @Honry @zqzhang please review and merge

**Comment by Honry** [comment]
LGTM :smile:

**Pull request merged**